docs/source/development/appextras.md

```diff
@@ -31,7 +31,7 @@ Or you can use the new key of `config.settings`, the `appExtras`. This is
 a list of registrations, each registration is an object with:
 
 - `match`: The `match` key is for objects compatible with [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath). It can be either a string or a [`match`](https://v5.reactrouter.com/web/api/match) object.
-- `ignore`: The `ignore` key is for routes compatible with the [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath) property, and ignores all the results in the `ignore` rule.
+- `ignore`: The `ignore` key is for routes compatible with the [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath) property, and ignores all the results in the `match` rule.
```

Comment:
I'd go one step further, assuming my following suggestion is correct. I don't know what a "rule" is in this context, but I do know what is a "key". Also using "ignore" as both a noun and a verb in the same sentence is very confusing.

```suggestion
- `ignore`: The `ignore` key is for routes compatible with the [react-router's `matchPath`](https://v5.reactrouter.com/web/api/matchPath) property, and excludes all the results from the `match` key.
```